docs/main/prerequisites.md

```diff
@@ -36,12 +36,29 @@ The following operating systems are **officially** supported:
 
     It is possible that Pi-hole will install and run on variants of the above, but we cannot test them all. If you are using an operating system not on this list you may see the following message:
 
-    ![non-supported OS warning](../images/OSWarning.png)
+    ```bash
+    [✗] Unsupported OS detected: Debian 16
```

Comment:
> `Debian 16`

I'm not sure this is intended, but the original image had `Debian 21` here.

Reply:
I know. I just needed a value that is not supported to be entered in `/etc/os-release` to trigger the warning. It happend I choose a different value this time. It does not matter, both are unsupported right now.